views/albums.hbs

```diff
@@ -0,0 +1,8 @@
+{{#each }}
+<div>
+
+    <h2>{{this.name}} </h2>
+    <input type="submit" value="View Albums">
```

Comment:
You do not need an input field here but you need a link to view the tracks.

`<a href="/tracks/{{ this.id }}">View tracks</a>`